commons/src/main/java/org/frankframework/util/StringUtil.java

```diff
@@ -157,9 +156,17 @@ public static String hideAll(String inputString, String regex) {
 	 * Else, all of it.
 	 */
 	public static String hideAll(String inputString, String regex, int mode) {
+		return hideAll(inputString, Pattern.compile(regex), mode);
+	}
+
+	/**
+	 * Hides the input string according to the given regex and mode.
+	 * If mode is set to 1, then the first half of the string gets hidden.
+	 * Else, all of it.
+	 */
+	public static String hideAll(String inputString, Pattern regex, int mode) {
```

Comment:
We still need both (overloaded) methods?

Reply:
For now, there are still callers that pass a string where I didn't feel it necessary to optimize.
That can perhaps come in a future PR, if the overloaded version with a `String regex` is no longer used.